variables.tf

```diff
@@ -15,22 +15,24 @@
  */
 
 variable "project_id" {
+  type        = string
   description = "The project ID to create the address in"
 }
 
 variable "region" {
+  type        = string
   description = "The region to create the address in"
 }
 
 variable "names" {
   description = "A list of IP address resource names to create.  This is the GCP resource name and not the associated hostname of the IP address.  Existing resource names may be found with `gcloud compute addresses list` (e.g. [\"gusw1-dev-fooapp-fe-0001-a-001-ip\"])"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
```

Comment:
enable_reverse_dns also missed, if we are adding explicit type, why not define it for them also and use boolean instean `ugly enable_reverse_dns != ""` constuctions

Reply:
Same

Reply:
Testing it now